eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/Eth2ReferenceTestCase.java

```diff
@@ -77,6 +77,13 @@ public abstract class Eth2ReferenceTestCase {
           .putAll(RewardsTestExecutorBellatrix.REWARDS_TEST_TYPES)
           .build();
 
+  private static final ImmutableMap<String, TestExecutor> EIP4844_TEST_TYPES =
```

Comment:
Updating Circle Contexts (Consensys#6660)

@joshuafernandes committed 3 weeks ago in commit ce90d1d177d6988e30de0c79ff1c9ec27b7b052a to the following file:

.circleci/config.yml

The changes made include replacing the "cloudsmith-protocols" context with "protocols-cloudsmith" and replacing the "dockerhub-quorumengineering-rw" and "dockerhub-opsquorum-dct" context in various filters with "protocols-dockerhub".